client/web/web.go

```diff
@@ -343,7 +358,7 @@ func (s *Server) serveAPI(w http.ResponseWriter, r *http.Request) {
 	switch {
 	case path == "/auth":
```

Comment:
I think you can actually just remove this case since it's handled above?

Reply:
oops, yeah... missed that one.  done